be/src/common/config.h

```diff
@@ -970,6 +970,9 @@ CONF_mBool(lake_enable_compaction_async_write, "false");
 CONF_mInt64(lake_pk_compaction_max_input_rowsets, "1000");
 // Used for control memory usage of update state cache and compaction state cache
 CONF_mInt32(lake_pk_preload_memory_limit_percent, "30");
+CONF_mInt64(lake_pk_compaction_max_sstable_versions, "5");
```

Comment:
lake_pk_index_sst_max_compaction_versions ?

Reply:
moved to next pr